signature/jws/envelope.go

```diff
@@ -57,8 +58,16 @@ func (e *envelope) Sign(req *signature.SignRequest) ([]byte, error) {
 		return nil, err
 	}
 
+	// parse payload as jwt.MapClaims
```

Comment:
Let's add a Reference URL to jwt.MapClaims in the comment.

Reply:
Added.